.github/workflows/build-macos.yml

```diff
@@ -0,0 +1,90 @@
+name: CI for tinygo-dev on macOS
+
+on: push
+
+jobs:
+  build-macos:
+    name: Build for macOS
+    runs-on: macos-11
```

Comment:
I'm a little bit worried that this will produce binaries that won't run on older MacOS systems.
Does anyone know how backwards compatible such binaries are? @dgryski maybe?

Reply:
I switched to `macos-10.15` for backward compat.